src/common/status.h

```diff
@@ -174,7 +174,7 @@ struct StringInStatusOr<T, std::enable_if_t<sizeof(T) < sizeof(std::string)>> :
   StringInStatusOr(StringInStatusOr<U>&& v) : BaseType(new std::string(*std::move(v))) {}  // NOLINT
   template <typename U, typename std::enable_if_t<!StringInStatusOr<U>::inplace, int> = 0>
   StringInStatusOr(StringInStatusOr<U>&& v)  // NOLINT
-      : BaseType((typename StringInStatusOr<U>::BaseType &&)(std::move(v))) {}
+      : BaseType((typename StringInStatusOr<U>::BaseType&&)(std::move(v))) {}
```

Comment:
```suggestion
      : BaseType((typename StringInStatusOr<U>::BaseType &&)(std::move(v))) {}
```

Don't change this line.

Reply:
Strange, I never touched this file. When I change it to how its supposed to be formatted and run the format script, it appends the && back to BaseType.

Reply:
@gloof11 The change was made by `./x.py format`. On my Fedora 38, I also have this behavior.

Reply:
@gloof11 You can undo the change manually and commit + push again.

Reply:
`./x.py format --clang-format-path clang-format-14`

I guess you're using clang-format-16.

Reply:
@infdahai Right. Before I upgraded Fedora from 37 to 38, the command `./x.py format` did its job the right way :)